### DISCUSSION TITLE
How to update React component on scroll?

### DISCUSSION THREAD

**Discussion by Ingimohamed** [codesandbox/codesandbox-client#5602] [Q&A]
In this example, I want the circular progress bar animates when scrolling down, and animate again when scrolling up.
I took the example from here -> https://www.kevinqi.com/react-circular-progressbar/

https://codesandbox.io/s/base-react-circular-progressbar-examples-forked-nofkj?file=/index.js:0-1867

I don't want to be repeating all the time just when scrolling only.

--- 

Edited by @smhutch — removed long code sample.

**Comment by smhutch** [comment]
@Ingimohamed could you share some more details of what you are trying to achieve. I'll try to point you in the right direction.

Is the intention that the progress bar is at 0% when at the top of the page, and 100% when scrolled to the bottom?
